examples/js/loaders/LDrawLoader.js

```diff
@@ -1174,8 +1174,6 @@ THREE.LDrawLoader = ( function () {
 
 									type = lp.getToken();
 
-									if ( ! parsingEmbeddedFiles ) {
```

Comment:
@makc I'm not sure about this one, I think `parsingEmbeddedFiles` could be `true`... 🤔

https://github.com/mrdoob/three.js/blob/dev/examples/js/loaders/LDrawLoader.js#L1254

Reply:
@mrdoob there is if ( parsingEmbeddedFiles ) { ...continue; } block above, so no.

Reply:
Oh I see!